### ISSUE TITLE
self destruction is not fun and frustrating. My solution/idea to ease the problem 

### ISSUE THREAD

**Issue by meiwud334** [endless-sky/endless-sky#3638]
I think the self destruction in the game to balance capping the drones is introducing grind and no fun gameplay. it feels arbitraty and unfun being fully reliand on RNG.

Solution I would like to discuss: Give the player some way to influence the destrcution rates and reduce exposure to extreme RNG.

Effreti could sell a special equipment: (something like Anti Selfdestruction Hacking Drone) which reduces the propability of self destrcution by C/(target mass) when used on boarding.
Example Numbers: outfitt space 3 Tons. Prize 5 million
Description: the bigger the target the less effect
This equippment could be unlocked by story, eg wanderer or just be sold from start by them (which i prefer as they currently have only the command center as unique item of interest).

when boarding a korath drone (or any ship with self destrcution rate <1) a new window will pop up: do you want to spend your Hacking Drones. yes or no
if yes how many:  chosse between 0-min (#installed,#minimum number needed to achieve 0 self destruct rate)

n=#Drone will be used up (removed from outfits installed) and the self destruction rate will be set to
new self destruction rate = old safe destruction rate -C x n/(target mass) >=0

Using Mass is a dirty quick fix to approxiamte how rewarding a ships is to cap and therefore how expensive it should be in terms of needed drones to be balanced (a 512 would need more drones than a model 8 to get perfect 0% blow up chances, this balances the holw drone thing).

Better: Alternatively to mass one could alter the self desctrution attribute to allow a second argument
"self destruct" .65 .1
the second number would tell how much one single drone would reduce the self destruct rate when used.
if left blank it is set to 0 and drones can not be used (no pop up window on boarding)

lets take a 512 as example with the new value set to .05 and the orginal self destruct rate
"self destruct" .80 .05
the player would need to use 20 drones (worth 20x5mil=100 mil) (20x3=60 outfit space) to achieve a self destruct rate of 0 and after that he needs to go and get new drones as these 20 are used up.

Lets discuss the andavtages and disadvantages of this idea:
pro: 
1.less frustrating RNG,
2. players than increase their chances by using proper equippment (buy with money), 
3. should be easy to programm, the players must not grind capping drones but has to grind money instead.
4. if the player feels like playing risky he can use fewer drones to reduce the explosion risk with some risk remaing with the consequeence that if it still blows up he lost some drones.
5. players that dont need to buy drones at all and can continue using RNG as is

con: 
1. drone ships are easier to cap when one can afford to buy the proper equippment. 
(See example below if balanced)
2. new unprecedented gameplay in form a pop asking for a number (other than taking a loan)

The pro's outweight the con's in my opinion. (Do you enjoy trying to cap a 512 for hours seeing them explode over and over when you finally managed to board a disabled one? Would you much rather pay 100 Mil to get a 100% chance of capping when you manage to board a disabled one)

Please tell me why my idea is terrible because i won't spend time pull requesting nothing that is destined to be not merged. Looking forward to your opinions. I hope i have presented the case clearly.

Note also the many post that focus about this issue befor me, many people dislike the RNG grind: http://steamcommunity.com/app/404410/discussions/0/364042703859367262/
https://steamcommunity.com/app/404410/discussions/2/224446614471344799/
https://steamcommunity.com/app/404410/discussions/0/135512625244825765/
https://groups.google.com/forum/#!topic/endless-sky/NZU1hWGpchc
https://groups.google.com/forum/#!topic/endless-sky/gKB7Y51WtpA
https://groups.google.com/forum/#!topic/endless-sky/gzBBpjCoZI0

**Comment by gunqqer** [comment]
As someone who has capped 10 512s I fully support something that can influence drone capture rates. It's annoying enough to find a single 512 that's disabled if you're letting them fight it out, or getting some on their own and disabiling them yourself.

**Comment by tehhowch** [comment]
Upcoming Wanderer story involves the drones - it's possible the player could make certain choices that make the drones purchasable. If there is such an outcome, I don't think a huge overhaul to self destruct rates - including entirely neutralizing them - will be needed.

I think the most amenable change will be to make the flavor outfits (node and transceiver) influence the specific faction, so long as both ships have the outfit installed, and also not the other kind (given that the two factions are at war, sensing the opponent AI in the boarding ship would certainly not lower the boarded ship's defenses.
At the same time, this change, while nice in lore, is not very flexible to other use cases without some rework of the outfits themselves.

In short - let's see how the remaining wanderer and Korath content play out before we alter the self destruct mechanic.

**Comment by geojak** [comment]
Good idea. Node And Receiver could Double/ halve (capped at 1 of course) the Self destruct Rates in the appropriate drones.

But then the game needs to check on every boarding if the target is a drone (when notes or receivers are installed, and it needs to track what is installed.)

Also the story involving drones is not directly involved with this issue in my opinion. As capping drones is done by the player mostly before finishing the wanderer story.

**Comment by ghoulavenger** [comment]
Considering that drones have no other defenses against capture mechanics I'm a bit skeptical of any changes to the way they are captured. That isn't to say I like self destruct, I find it extremely annoying and have resigned myself to NOT using their ships in my fleet. The problem is that it is perfectly possible to acquire entire fleets of these ships extremely easily if it is made "too easy". So if you wanted to get rid of or reduce the self destruction chance you really need something else to balance it, like actual crew outfits that repel boarders (which isn't without it's own problems).

I do look forward to what MZ has planned for the rest of the Wanderer story, and if there are any new wrinkles to work on because of it.

**Comment by Tadrix** [comment]
For now, how about just lowering self destruct rates?

The current situation:

self destruct | chance of docking | chance of capturing
--- | --- | ---
.8 | 20% | 4%
.79 | 21% | 4,41%
.77 | 23% | 5,29%
.74 | 26% | 6,76%
.7 | 30% | 9%
.65 | 35% | 12,25%
.6 | 40% | 16%

I propose to alter self destruct values so that current values represent self destruct chance on capture attempt:

current self destruct | proposed self destruct
--- | ---
.8 | 0.55
.79 | 0.54
.77 | 0.52
.74 | 0.49
.7 | 0.45
.65 | 0.41
.6 | 0.37

**Comment by tehhowch** [comment]
@tadrix That's a huge increase for ships which do not consume crew to capture, and thus are limited only by how much time you want to put in (and by the DPS you can withstand).
I would much prefer hacking drones over that method.

**Comment by Tadrix** [comment]
Hacking drones would be nice. The proposed solution was "for now". Also, the chance to self destruct is too high if you only want to plunder them. So maybe we could bypass the check on docking attempt instead of what I proposed a post ago.

**Comment by tehhowch** [comment]
It would have to be outfit space to be a reliable way to have it on your flagship. A concept similar to the existing ammo containers would work

**Comment by Tadrix** [comment]
Thing is, you need that outfit space to survive if you are not going in with a large fleet (drone fights can be brutal). So either you'd need drones with negligible outfit space requirements (e. g. 0.04 t per drone), or a holder (like ammo storage) and drones as ammunition (again, with negligible mass).

**Comment by KiLEdEnNis** [comment]
I did like the feeling of accomplishment when I captured my 1st 349 (in my 1st playthrough after a long while), it feels rewarding basically
on the other side, I do find the self-destructing mechanic annoying as well, but I don't think something like this is the way to fix it

first, I think that boarding and attempting to capture a ship should have 2 independent self-destruct chances (like boarding gives a 20% self-destruct rate, attempting to capture has an 80% chance)
this would also make boarding to plunder a bit less annoying
second, if the first isn't enough, I think a counter to self-destruction shouldn't reduce the chance by a flat number, I'd rather have an outfit that reduces it by a percentage, and the reduction moves to an asymptote
so basically: stacking said outfit would be less effective the more ya have, and ya can't reduce a ship's self-destruct chance by, for example 50%
this should also avoid said outfits becoming OP

**Comment by jafdy** [comment]
But then people will complain that they used some ridiculous number of what ever it is and the ship still self destructed. 
I would quite like to see some kind of automated defence so that you will lose crew even to a drone (could be something as simple as powerful doors closing on people).

**Comment by jostephd** [comment]
A drone that gets disabled could self-destruct before the player boards it. There are various ways to do this, for example, the drone could have 85% to self-destruct at the 15 seconds mark after being disabled and 50% to self-destruct at the 30 seconds mark, then if it survives more than 30 seconds it's a shipwreck that can be boarded without risk. This way the player can either rush to board the drone as soon as it's disabled, or wait and hope that this drone is the rare one that doesn't self-destruct, and that it won't be destroyed by crossfire.

**Comment by jostephd** [comment]
A drone that's about to get disabled would turn itself into a kamikaze, wouldn't it?

**Comment by KiLEdEnNis** [comment]
that's just another annoying idea that'd feel clunky when the ships may self-destruct at a set time after being disabled
and what if allies are about to repair it? or allies may win the fight and will repair them afterwards? it makes 0 sense for drones to self-destruct when they get disabled

**Comment by jostephd** [comment]
The lore of this is that drones self-destruct so the enemy faction doesn't have a chance to plunder them. Drones don't have a "leave no man behind" rule; if a drone got disabled in action, let it die, another one will be come off the production line any minute now. Better for the drone to be lost than to risk that the enemy will plunder it.

**Comment by jafdy** [comment]
On the whole a better solution is needed. And arguing over what non-so-good solution is better isn't going to fix anything.

**Comment by KiLEdEnNis** [comment]
> The lore of this is that drones self-destruct so the enemy faction doesn't have a chance to plunder them. 

why not self-destruct once an enemy boards them then?
also; how would a robot ship even plunder another one? the factories have robots that create them, they don't have robots onboard to do such things
as in better to not self-destruct as long as ya don't get boarded since ya may get repaired and live

as for what Jafdy suggested to be paired with the idea of lowering self-destruct chances
> I would quite like to see some kind of automated defence so that you will lose crew even to a drone

having to somehow fight the automaton upon boarding wouldn't be unrealistic; they could have internal defenses (although lore-wise it may seem a bit odd)
and this would enable lower self-destruct chances since boarding them wouldn't become OP

**Comment by jostephd** [comment]
> why not self-destruct once an enemy boards them then?

The self-destruct mechanism is not perfect. 9 times out of 10 it does what it's supposed to do but sometimes it doesn't, because it was damaged in combat or by space seawater or something.

**Comment by ghoulavenger** [comment]
> I would quite like to see some kind of automated defence so that you will lose crew even to a drone (could be something as simple as powerful doors closing on people).

This goes along with my suggestion:
>  like actual crew outfits that repel boarders 

And I think you could make it work. You would need an outfit that added a mechanical crew attribute that was consumable and not plunderable. Making it consumable also makes it more reasonable, since players wouldn't be able to use these "fake crew" to their own advantage.

> A drone that gets disabled could self-destruct before the player boards it.

I don't like this one as much. It plays into the same thing as crossfire basically. You wouldn't notice it at all, you'd just be left with capturable ships if you were fast enough to get to them. Also with only one check you're already significantly increasing the chances of capturing their ships (they self destruct on board and capture attempts).

**Comment by Bladewood** [comment]
If we make it such that *all* automata self-destruct shortly after being disabled, it could constitute a nerf in a different way: the player would be in a rush to repair disabled escort or friendly automata before it is too late.

**Comment by Tadrix** [comment]
This would be frustrating in it's own way.

Larger drones could have some type of internal defenses indeed. Not Model 8's, but capital ships. Small drones could attempt going kamikaze instead.

(KIV349, M256, M512 — well defended; TF71–L, TF78–O, TF109, M64, M128 — less defended; MPT53, M16, M32 — very lightly defended; FL14, FO27, M8 — attempts to go kamikaze on 50% hull, no defenses).

Lorewise, if there are autorepair bots, then while unable to do their work (b/c the drone is disabled) they could change to combat-ready mode (instead of things like doors or weapons which logically shouldn't be inside the drone).

**Comment by Hadron1776** [comment]
Or just take advantage of the fact that drones have no atmosphere (because no pilot) and require that the player bring a sort of EV suit in order to cap it or plunder anything.

**Comment by hellminister** [comment]
if the drones have auto repair bots, they could self repair themselves without needing an ally, making them dangerous to board when they reactivate... ;)

**Comment by geojak** [comment]
I would like to point out that ops idea requires to player to actually work/invest for better chances. He earns them for buying special outfits and bringing them along in outfit space.

I think that is a general concept that should be included in any way of changing the current situation.

Also making reasoning nodes etc having a role to play would be cool so they are not completely useless except for lore.

**Comment by starcatter** [comment]
I've tried adding a mechanic where installing reasoning/control nodes in the capturing ship would reduce the drones' self destruct chance:
```c++
	const double Ship::getSelfDestructOddsForAttacker(const std::shared_ptr<const Ship> &attacker){
		double chance = Attributes().Get("self destruct");

		if( GetGovernment()->GetName() == "Kor Mereti" ){
			double nodes = attacker->Attributes().Get("mereti control");
			chance /= 1+(nodes/5.);
		} else if( GetGovernment()->GetName() == "Kor Sestor"  ){
			double nodes = attacker->Attributes().Get("sestor control");
			chance /= 1+(nodes/5.);
		}

		return chance;
	}
```
I added "mereti control" and "sestor control" attributes to the nodes and hooked up this method to checks when boarding/capturing. I realize this is rather weak, story wise (I mean, how would that even work? Can it be implied the Kor outfitter rewires the nodes into some kinda jammers?), but for experimental purposes it's pretty close to a scenario where the nodes would be transformed somehow or the player got some other means of reducing the self destruct chance.

The result was that getting the first few nodes was a bit tedious, but once I installed around 40 capturing success was pretty much guaranteed. Adding some outfits expansions and giving up some weapons allowed for a Kestrel loadout that could capture both kinds of drones. This together with the captured ships quickly adding up to a force that could easily disable even sizable drone fleets quickly lead to a highly unbalanced game.

The most unbalancing aspect here was the fact I could farm the drones indefinitely in systems with uninhabited planets, resulting in a growing fleet of auto-repairing ships at no resource cost. While the 100% capture chance accelerated that process, any kind of capture chance that wouldn't be as frustrating as the current ones would eventually lead to this outcome. I suggest there be some sort of "inhibitor" that needs to be installed on a capturing ship, either a special weapon that needs to be fired at the disabled drone to greatly reduce the self-destruct chance or an outfit that is expanded upon docking with a drone. They might simply be sold at the Kor Efret outfitter after some quest chain or produced by the Kor Efret engineers the player meets during the Wanderer story. Either way it ought to be expensive, heavy and preferably require a bunch of nodes from captured drones to produce. I don't see a need for a high failure chance once a "inhibitor" was expanded on an enemy ship, there's enough risk involved in keeping the captured ship alive. Effectively limiting the number of captured drones per trip to 5 would help a lot. Additionally, there might be a cost to somehow convert the drones before they become fully functional, for example making them behave like ships with insufficient crew right after capture.

IMO there's a good moment during the Wanderer campaign where the Kor Efret engineers might offer the player some sort of experimental equipment which would allow for capturing the drones, which would ensure there's no easy way of obtaining a drone fleet early in the game. Alternatively, they might offer to help as soon as the player lands on a Kor Efret world with a captured drone.

A more complex solution I have in mind is making all Kor drones self destruct soon after being disabled, except for the Kor Sestor carrier-based fighters/drones. The player might be tempted to capture one of these. Upon success, they get a message that there's no way of controlling the ship they captured. If they have a suitable carrier in their fleet with a free bay, they can choose to pick up the captured drone, otherwise they can try boarding it again with 100% success chance, to strip outfits etc.

Bringing a captured fighter to a Kor Efret world would trigger a mission to bring it to the Kor Efret engineers, who would provide the player with tech which would prevent the drones from self destructing after being disabled, somehow also working on Kor Mereti drones. That would start a quest chain leading to the development of some sort of hardware allowing for easier capture, and control of Kor drones.

Another issue with the captured drone fleet was that it had no upkeep cost. Crew salaries is what keeps  players from flying ridiculously oversized fleets, or at least from having them active all the time. With drones costing nothing, there's no real motivation to limit fleet size, which might lead to players hitting technical limitations (as in, low frame rates due to hundreds of ships on screen). IMO some sort of drone control stations might be a reasonable solution, each station requiring one crew to control, taking 5 outfit space and providing 20 "drone control". Carrier-based drones would require 1 control, Model 512/Kar Ik Vot 349 would require 20. I'm not sure if the stations should be limited to the flagship.

I think these two mechanics, once refined and balanced, should help keep the scale of captured drone fleets within reasonable limits, at least for most players. I can help coding engine features required to make this work.

**Comment by geojak** [comment]
That was a great read. Good constructive work and we thought trough!

How about a repeatable mission from the effret offered on landing like the hai jump drive, which allows to exchange 10 reasoning nodes for one self destruct disabling outfit, launch able as described by you above.

Then players still have to go through the tedious self destruction to gain at least some notes but can earn a safe shot from it. Wouldn't speed the process of farming drones too much but allows players to control their chances a bit better.

Also the drones control center is a great idea. Maybe expand the existing control center for this purpose?

**Comment by zorodc** [comment]
Story-wise, I've always thought the justification for why the Korath Automatons don't always self-destruct, is that although they are programmed to do so to avoid their technology and supplies from falling into enemy hands, a disabled autonomous ship may have had components employed in executing this apoptosis be damaged, whereupon it is disabled.

However, when one docks with a disabled automaton - either the ship self-destructs, or one can plunder it. Try to capture it, however, and it may or may not self-destruct again! I would suggest, story-wise, that the second round of self-destruction seems implausible, as in theory the self-destruction mechanism is already damaged - why would it suddenly begin to function at so opportune a moment? With manned ships, this behaviour makes sense - capturing a ship presently implies the death of all those on board, and one could easily think that a group of individuals fearing their own death might wish for it to occur on their own terms - denying any profit to their attackers, but otherwise not acting if only their ship is plundered and their lives left relatively secure. It is also less obnoxious when this occurs for manned crafts, because manned crafts self-destruct far less frequently, when compared to the automatons. The automatons potentially self-destructing on attempt at capture also doesn't do much in the way of balance - as one can merely take the route of plundering the outfits, and fitting a powerful ship with them, and receive a mostly equivalent reward in all ways except cost. However, many players do not enjoy doing this, as it it time consuming. I don't believe it to be justifiable, in the case of the automatons, to impart this second round of potential self-destruction - the player faces a rather annoying dilemma, and must choose between potentially loosing anything of value (not plundering and trying to take the ship), and wasting a bit of time configuring a different ship with the plundered outfits.

I wanted to suggest something minimally intrusive, and so I suggest that automatons bear the potential to self-destruct only initially, when they are docked with, making capture an otherwise free option. I feel this is not unbalanced because a recently captured automaton ship is somewhat likely to be destroyed subsequent to its capture, unless one is either careful or lucky. The positive qualities of this suggestion is that, firstly - as far as I can estimate, they make the behaviour of the automatons more logical, secondly - the balance of self-destruction in general general does not have to be manipulated, - and thirdly, it doesn't require the addition of anything to the story, nor the addition of items that require balancing considerations. In fact, I would suspect that removing additional factors in the consideration of automaton capturing would make balancing contingent upon fewer factors, making balancing easier. I apologize for the length of this posting.

Would, perhaps, something like this be sufficient to satisfy the desires of those frustrated by the indeterminism encountered in capturing automata ships? Are there any thoughts?

**Comment by Nescio0** [comment]
// A ship that self-destructs checks once when you board it, and again
		// when you try to capture it, to see if it will self-destruct. This is
		// so that capturing will be harder than plundering.

Yes, I fully realize the above is intentional, however, I also think applying it only once could be better and certainly less frustrating. Of course, the "self destruct" chance would have to be raised in compensation.
That special outfit suggestion sounds unnecessarily complicated to me.

**Comment by endless-sky** [comment]
The self-destruct check is done twice because capturing a ship is *much* better than just pulling some outfits out of it, especially if it's a ship that has no crew requirement (meaning there's no limit on how many you can capture at once).

**Comment by tehhowch** [comment]
We could ease up on self-destruct rates if we add crew attrition when capturing automatons. This would also end infinite-farm sorties, as the player would need to return to a planet to recrew eventually. The difficult part is figuring out a portable "you lost crew somehow" message.

**Comment by jostephd** [comment]
How about if once you board an automaton, you have to leave a few crew members behind to work on mending it. Then after a few minutes, either it becomes un-disabled and joins your fleet as an escort, or it becomes un-disabled and turns into a pirate. (Yes, yes, automatons don't have cockpits, but I'm sure a couple of engineers in an alien spaceship with no adult supervision can set up some sort of cockpit out of two used gamepads and a micro-USB cable.)

**Comment by Zitchas** [comment]
Along the same lines, this might be a good situation to make some of those command centers a purpose. It might not be worth the extra coding, but it'd be interesting if command centers were a special outfit that you could install in ships via boarding action. So if you had one in your cargo bay and boarded an automaton ship, you could install that module and transfer a crew member as a means of bypassing the self-destruct. They'd still have to get passed the initial "on boarding" self-destruct, but it would allow a way to bypass the second one.

(Unless the player has some lore way of tapping into the automatic control grid, I'd say that the player should be required to have a control center installed on any automata ship in their fleet in order to get it off the ground at all. That would eliminate the zero crew costs problem with automata fleets.)

**Comment by Tadrix** [comment]
Could we have self-destruct checks only on capture attempts?

**Comment by KiLEdEnNis** [comment]
~separate boarding and capping self-destruct chances~

**Comment by VonZorch** [comment]
@zorodc 
" I would suggest, story-wise, that the second round of self-destruction seems implausible, as in theory the self-destruction mechanism is already damaged - why would it suddenly begin to function at so opportune a moment?"
The bane of an electronics technicians life, the intermittent fault. Something as simple as a cable loosened during combat, when docking it's not in contact so no boom, but the vibration from docking puts it back in contact so when you try to board the signal goes through, BOOM.

**Comment by kestrel1110** [comment]
> I've tried adding a mechanic where installing reasoning/control nodes in the capturing ship would reduce the drones' self destruct chance:
> 
> ```c++
> 	const double Ship::getSelfDestructOddsForAttacker(const std::shared_ptr<const Ship> &attacker){
> 		double chance = Attributes().Get("self destruct");
> 
> 		if( GetGovernment()->GetName() == "Kor Mereti" ){
> 			double nodes = attacker->Attributes().Get("mereti control");
> 			chance /= 1+(nodes/5.);
> 		} else if( GetGovernment()->GetName() == "Kor Sestor"  ){
> 			double nodes = attacker->Attributes().Get("sestor control");
> 			chance /= 1+(nodes/5.);
> 		}
> 
> 		return chance;
> 	}
> ```
> 
> I added "mereti control" and "sestor control" attributes to the nodes and hooked up this method to checks when boarding/capturing. I realize this is rather weak, story wise (I mean, how would that even work? Can it be implied the Kor outfitter rewires the nodes into some kinda jammers?), but for experimental purposes it's pretty close to a scenario where the nodes would be transformed somehow or the player got some other means of reducing the self destruct chance.
> 
> The result was that getting the first few nodes was a bit tedious, but once I installed around 40 capturing success was pretty much guaranteed. Adding some outfits expansions and giving up some weapons allowed for a Kestrel loadout that could capture both kinds of drones. This together with the captured ships quickly adding up to a force that could easily disable even sizable drone fleets quickly lead to a highly unbalanced game.
> 
> The most unbalancing aspect here was the fact I could farm the drones indefinitely in systems with uninhabited planets, resulting in a growing fleet of auto-repairing ships at no resource cost. While the 100% capture chance accelerated that process, any kind of capture chance that wouldn't be as frustrating as the current ones would eventually lead to this outcome. I suggest there be some sort of "inhibitor" that needs to be installed on a capturing ship, either a special weapon that needs to be fired at the disabled drone to greatly reduce the self-destruct chance or an outfit that is expanded upon docking with a drone. They might simply be sold at the Kor Efret outfitter after some quest chain or produced by the Kor Efret engineers the player meets during the Wanderer story. Either way it ought to be expensive, heavy and preferably require a bunch of nodes from captured drones to produce. I don't see a need for a high failure chance once a "inhibitor" was expanded on an enemy ship, there's enough risk involved in keeping the captured ship alive. Effectively limiting the number of captured drones per trip to 5 would help a lot. Additionally, there might be a cost to somehow convert the drones before they become fully functional, for example making them behave like ships with insufficient crew right after capture.
> 
> IMO there's a good moment during the Wanderer campaign where the Kor Efret engineers might offer the player some sort of experimental equipment which would allow for capturing the drones, which would ensure there's no easy way of obtaining a drone fleet early in the game. Alternatively, they might offer to help as soon as the player lands on a Kor Efret world with a captured drone.
> 
> A more complex solution I have in mind is making all Kor drones self destruct soon after being disabled, except for the Kor Sestor carrier-based fighters/drones. The player might be tempted to capture one of these. Upon success, they get a message that there's no way of controlling the ship they captured. If they have a suitable carrier in their fleet with a free bay, they can choose to pick up the captured drone, otherwise they can try boarding it again with 100% success chance, to strip outfits etc.
> 
> Bringing a captured fighter to a Kor Efret world would trigger a mission to bring it to the Kor Efret engineers, who would provide the player with tech which would prevent the drones from self destructing after being disabled, somehow also working on Kor Mereti drones. That would start a quest chain leading to the development of some sort of hardware allowing for easier capture, and control of Kor drones.
> 
> Another issue with the captured drone fleet was that it had no upkeep cost. Crew salaries is what keeps players from flying ridiculously oversized fleets, or at least from having them active all the time. With drones costing nothing, there's no real motivation to limit fleet size, which might lead to players hitting technical limitations (as in, low frame rates due to hundreds of ships on screen). IMO some sort of drone control stations might be a reasonable solution, each station requiring one crew to control, taking 5 outfit space and providing 20 "drone control". Carrier-based drones would require 1 control, Model 512/Kar Ik Vot 349 would require 20. I'm not sure if the stations should be limited to the flagship.
> 
> I think these two mechanics, once refined and balanced, should help keep the scale of captured drone fleets within reasonable limits, at least for most players. I can help coding engine features required to make this work.

So maybe make the Node and Transceiver have the attributes, and then make them not stack. Then make it so each Node/Transceiver will make it 20% easier to capture, or maybe some other value. Just a suggestion though.

**Comment by pepoluan** [comment]
Maybe add a new component, say "Hacking Station" (bad name, I know).

Each Hacking Station reduces chance of self-destruct by 20%, but compounded. So if you install 4 of them, you don't reduce `.8` to `-.2`, rather you reduce it to `.8 * (80% ^ 4)` = 0.33`. Still chance of self-destruct, but much smaller.

Then let's give the Remnant capital ships an innate Hacking power of 30% 😉

**Comment by asl97** [comment]
My proposal would be to use the energy level for the self-destruction chance, 100% above a certain level (in absolute value so lower end ships are easier to drain below said level but higher end ship are way harder to drain) but lower drastically to maybe even 0% at no power.

The Ion cannon does too much damage to drain the energy with overwhelming force/fleet, so one would have to draw out the battle or wait till they obtain better ion weaponry like the Ke'Hat Nullifier or EMP Deployer.
